common/src/main/java/com/datastrato/gravitino/json/JsonUtils.java

```diff
@@ -212,7 +212,7 @@ String convert(JsonNode element) {
     }
   }
 
-  private static volatile ObjectMapper mapper = null;
```

Comment:
This is correct without issues. But I prefer using lazy Singleton like https://blog.csdn.net/iteye_20954/article/details/81342188 . I feel that it's more elegant. You can raise another pr to fix this.

Reply:
okya, i will take a closer look at the problem 1935 and try to fix, thx for your remind.